docs/components.md

```diff
@@ -109,6 +109,18 @@ useEffect1(effect, [|dep|])
 
 However, as the length of the array is not specified, you could pass an array of arbitrary length, including the empty array, `[||]`.
 
+> *NOTE*: When using `useEffectN` for `N` greater than 1, do not try to gather like types into arrays. Instead, leave each item separate in the tuple. 
```

Comment:
I would explain this a bit differently:

> Using useEffectN1/N2/N3/... make sure you don't wrap the dependencies in arrays. Since tuples are represented in arrays in JavaScript it would trick React into compare arrays by reference (which is often undesired).

> Use the tuple given one dependency per item, even when those types are the same.